k8s/overlays/nerc-shift-0/stf/stf-object.yaml

```diff
@@ -80,7 +77,7 @@ spec:
       web:
         enabled: false
   clouds:
-    - name: nerc-overcloud
```

Comment:
cloud name had to be changed because STF now enforces: '^[a-zA-Z0-9]{1,10}$'

this name is added to prometheus data and is likely to have consequences to grafana views.
example:  service="default-nerc-overcloud-coll-meter"